### PR TITLE
removed path attribute from class RoleSettings class

### LINKED ISSUE #259: Task: remove "paths" attribute from "RoleSettings"
### What is the task about?

We no longer need the `paths` attribute from https://github.com/vmware/repository-service-tuf-api/blob/94f4f4a3af7dc56c71342a3521b426dd8c27876c/repository_service_tuf_api/bootstrap.py#L124.
This change should be on top of the `feature/roles-simplification` branch.

### Parent feature

_No response_

### References

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by deepgohil**
removed path attribute from class RoleSettings class in file repository_service_tuf_api/bootstrap.py

Closes #259

### CODE REVIEW

**Comment by MVrachev** [comment]
Thank you @deepgohil for your first contribution!

First, you should remember to [sign off](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff) your commits (basically using the `-s` with the `git commit command`).
It is essential to agree with the [project's DCO](https://repository-service-tuf.readthedocs.io/en/latest/devel/contributing.html#dco)

Second, the error message you see `Error: Process completed with exit code 255.` here https://github.com/vmware/repository-service-tuf-api/actions/runs/4163657578/jobs/7205257315 doesn't give you the full information.
If you scroll above you will see that: `test: FAIL ✖ in 39.48 seconds and you can check the tests run by clicking `tox: test` button:
image

this will expand information about the executed tests.
There you can see errors in some tests.
Basically, the problems are that there are places where you create a `RoleSettings`object and try to access the `path` attribute which you just removed.
Try fixing this locally and if you want to run the tests locally you should have installed an environmental variable and installed our dependencies (see https://github.com/vmware/repository-service-tuf-api#installing-project-requirements) and then you can run `tox -e test` to check if you have fixed the tests.
If you still need help, don't hesitate to ask.

**Comment by MVrachev** [comment]
Also, I don't want to forget:
this change should be aimed against the `feature/roles-simplification` branch.
If you don't mind I will change the target branch because this is part of a bigger feature we develop in a separate branch.

**Comment by MVrachev** [comment]
I opened a similar pr to yours which you can use as help if you want: 
https://github.com/vmware/repository-service-tuf-api/pull/262

Also, if you want to message as and get a faster response you an ask us at the `[#repository-service-for-tuf](https://cloud-native.slack.com/archives/C047L55314N)` channel on the [CNCF Slack Workspace](https://slack.cncf.io/).

**Comment by MVrachev** [comment]
@deepgohil I just realized that I will need to remove the `path` attribute into another pr https://github.com/vmware/repository-service-tuf-api/pull/262.
The idea is that a new version of the CLI will be released and we need to be compatible with it.

I can help you with choosing another good-first-issue if you want.

**Comment by MVrachev** [comment]
Closing this pr (thanks @deepgohil again for your contribution), but I needed this change merged soon.
This pr is surpassed by #262
